meetings/2020-04-08.md

```diff
@@ -0,0 +1,93 @@
+﻿# Node.js Technical Steering Committee (TSC) Meeting 2020-04-08
+
+## Links
+
+* **Recording**: https://youtu.be/2om4bZws47A
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/846
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Beth Griggs @BethGriggs (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* James Snell @jasnell (TSC)
+* Matheus Marchini @mmarchini (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Rich Trott @Trott (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Sam Roberts @sam-github (TSC)
+* Shelley Vohr @codebytere (TSC)
+* Tobias Nießen @tniessen (TSC)
+
+## Agenda
+
+
+### Announcements
+
+
+* Nominating @Himself65 as a collaborator [#32441](https://github.com/nodejs/node/issues/32441)
+
+* Nominating @puzpuzpuz as a collaborator [#32555](https://github.com/nodejs/node/issues/32555)
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+
+### nodejs/node
+
+* quic: part 3... initial quic implementation [#32379](https://github.com/nodejs/node/pull/32379)
+* quic: part 2... ngtcp2 and nghttp3 [#32378](https://github.com/nodejs/node/pull/32378)
+* quic: part 1... quic-modified openssl [#32377](https://github.com/nodejs/node/pull/32377)
+  * Rich - these are an FYI, not time critical, let's discuss concerns raised when James is in the
+    call ?
+  * Myles, time critical part is whether it makes it into 14.x. Rich this can go into a later 14.x
+    version since it is Minor so possibly not time sensitive.

```

Comment:
```suggestion
  * Myles: time critical part is whether it makes it into 14.x.
  * Rich: this can go into a later 14.x version since it is Minor so possibly not time sensitive.
```